.travis.yml

```diff
@@ -25,6 +26,10 @@ matrix:
     - os: linux
       python: "3.6"
       env: TEST_UNIT=0 TEST_STYLE=1
+    - os: linux
+      python: "3.6"
+      arch: arm64
+      env: TEST_UNIT=0 TEST_STYLE=1
```

Comment:
You can remove this one: for style tests the machine should not matter.

Reply:
Sure...Will modify it.